admin-dev/themes/default/template/helpers/list/list_content.tpl

```diff
@@ -109,7 +109,7 @@
 							{/if}
 						{/if}
 					{elseif isset($params.type) && $params.type == 'price'}
-						{if isset($params.currency)}
+						{if isset($params.currency) && is_string($params.currency)}
```

Comment:
How about we extended the condition with a check that the `$tr` array actually contains this key? Something like

```smarty
{if isset($params.currency) && is_string($params.currency) && array_key_exists($params.currency, $tr)}
```

Reply:
Very good point. I haven't tested it, but it makes a lot of sense. Why haven't you taken it?

Reply:
I thought about it a little bit more. It probably shouldn't be responsibility of `view` to validate input. I think it would be better to do such validation in php, in `HelperCore`, `HelperList` or `HelperForm`. We can do it much easily there, maybe conditionally on debug mode settings. We can much better report on source of the problem (what controller is the culprit, etc...)